src/app/public/modules/lookup/lookup-show-more-modal.component.ts

```diff
@@ -0,0 +1,331 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  EventEmitter,
+  OnDestroy,
+  OnInit,
+  Output
+} from '@angular/core';
+
+import {
+  SkyDataManagerState,
+  SkyDataViewConfig,
+  SkyDataManagerService
+} from '@skyux/data-manager';
+
+import {
+  SkyModalInstance
+} from '@skyux/modals';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  debounceTime,
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyLookupSelectMode
+} from './types/lookup-select-mode';
+
+import {
+  SkyLookupShowMoreContext
+} from './types/lookup-show-more-context';
+
+@Component({
+  selector: 'skyux-lookup-show-more-modal',
+  templateUrl: './lookup-show-more-modal.component.html',
+  styleUrls: ['./lookup-show-more-modal.component.scss'],
+  providers: [SkyDataManagerService],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyLookupShowMoreModalComponent implements OnDestroy, OnInit {
```

Comment:
Do we want this class showing up in the docs? Need an `@internal` JSDoc?

Reply:
Not relevant now due to the modal work split up.